TradeItIosTicketSDK2/TradeItSession.swift

```diff
@@ -55,10 +55,10 @@ public class TradeItSession {
         if let authenticationResult = TradeItResultTransformer.transform(targetClassType: TradeItAuthenticationResult.self, json: json) {
             self.token = authenticationResult.token
             return authenticationResult
-        } else if let securityQuestionResult = TradeItResultTransformer.transform(targetClassType: TradeItSecurityQuestionResult.self, json: json) {
+        } else if let securityQuestionResult = authenticationResult as? TradeItSecurityQuestionResult {
```

Comment:
Where does this get transformed now?

Reply:
https://github.com/tradingticket/TradeItIosTicketSDK2/blob/2e50f46fb87183de63d5fc44e7475f9ef5363405/TradeItIosTicketSDK2/TradeItConnector.swift#L78